domains/ysandeepkumarreddy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ysandeepkumarreddy",
+        "email": "sandeepkumarreddy2002@gmail.com"
+    },
+    "record": {
+        "CNAME": "d18bjwu9yqnffu.cloudfront.net"
```

Comment:
Inaccessible site, please give a preview of the site (though if you don't want this file then you can delete it)